presidio-image-redactor/presidio_image_redactor/dicom_image_redactor_engine.py

```diff
@@ -43,6 +45,8 @@ def redact(
         :param padding_width: Padding width to use when running OCR.
         :param crop_ratio: Portion of image to consider when selecting
         most common pixel value as the background color value.
+        :param redact_approach: What approach to use when redacting
```

Comment:
its a bit weird that the default is not 'default'. Maybe choose different terminology